src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health-pie/health-pie.component.scss

```diff
@@ -4,6 +4,17 @@ $canvas-width: 100%;
 $canvas-height: 100%;
 
 .chart-container {
+  --color-red: #c9190b;
+  --color-blue: #06c;
+  --color-orange: #ef9234;
+  --color-yellow: #f6d173;
+  --color-magenta: #009596;
+  --color-green: #7cc674;
+  --color-gray: #ededed;
+  --color-light-blue: #519de9;
+  --color-light-yellow: #f9e0a2;
+  --color-cyan: #73c5c5;
+  --color-purple: #3c3d99;
```

Comment:
As @ishanrai05 recently delivered a PR for removing literal hexcolors from CSS, could just export as CSS vars the 11 SASS vars that we need for this?
```scss
--color-red: $red;
--color-orange: $orange;
...
''
```
![image](https://user-images.githubusercontent.com/37327689/90041163-32615c00-dcc9-11ea-8cdf-e4c6aa1fc4de.png)

Reply:
Done.